Studio-Lovelies/GG-JointJustice-Unity
# Feature/121 studio lovelies unity splash

## Summary

Implements the Studio Lovelies splash screen. A requested feature was that it randomly played an alternate splash, so the feature is implemented in its own scene, as opposed to Unity's splash screen options.

## Before/after screenshots and/or animated gif

![splash3](https://user-images.githubusercontent.com/67964179/152744762-024d776c-c4fe-4e06-9585-6e123c4876ea.gif)

## Testing instructions

Play the "Splash" scene. Does the splash play? Does it automatically continue to the menu scene. Export the game, and check the splash plays properly in the exported build.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource: 
  
closes #121
